Assets/Script/Gameplay/TextureManager.cs

```diff
@@ -102,9 +161,11 @@ private void UpdateFFT(NativeArray<byte> pixelData)
                 // set spectrum data in the first row
                 pixelData[i] = (byte) Math.Round(normalized);
                 // waveform data in the second row
-                var wave = _wave[i] * (1.0f - waveSmoothingFactor) + _prevWave[i] * waveSmoothingFactor;
+                // I can't find any wave smoothing in the FF or ShaderToy code, so this is commented out
+                // var wave = _wave[i] * (1.0f - waveSmoothingFactor) + _prevWave[i] * waveSmoothingFactor;
```

Comment:
I don't actually remember if it has it either, so we can just remove/cleanup that completely if you feel its unnecessary